cluster/cluster_impl/failsafe_cluster.go

```diff
@@ -31,7 +31,9 @@ func init() {
 	extension.SetCluster(failsafe, NewFailsafeCluster)
 }
 
-// NewFailsafeCluster ...
+// NewFailsafeCluster returns failsafe cluster instance
+//
+// Failure of security, anomalies, directly ignored. Usually used to write audit logs and other operations.
```

Comment:
// NewFailsafeCluster returns an failsafe cluster instance.
//
// Failure of security, anomalies, directly ignored. Usually it is 
//  used to write audit logs and other operations.